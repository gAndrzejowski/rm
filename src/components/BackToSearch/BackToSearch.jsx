// @flow
import React from 'react';
import { Link } from 'react-router-dom';

const BackToSearch = () => (
  <Link to="/" href="/">
    <button>Search</button>
  </Link>
);
export default BackToSearch;
