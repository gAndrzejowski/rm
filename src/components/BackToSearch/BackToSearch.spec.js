import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import BackToSearch from './BackToSearch';

describe('back to search component', () => {
    it('renders correctly', () => {
        const renderer = new ShallowRenderer();
        renderer.render(<BackToSearch/>);
        expect(renderer.getRenderOutput()).toMatchSnapshot();
    });
});