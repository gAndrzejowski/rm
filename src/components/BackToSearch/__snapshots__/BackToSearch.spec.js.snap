// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`back to search component renders correctly 1`] = `
<Link
  href="/"
  replace={false}
  to="/"
>
  <button>
    Search
  </button>
</Link>
`;
